environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -213,7 +213,7 @@ resources:
                 fi
               fi
             fi
-            if [[ `echo $role |awk -F"-" '{print $2}'` == "novacompute" || `hostname |awk -F"-" '{print $2}'` == "contrailtsn" ]]; then 
+            if [[ `echo $role |awk -F"-" '{print $2}'` == "compute" || `hostname |awk -F"-" '{print $2}'` == "contrailtsn" ]]; then
```

Comment:
Here $role could be either taken from the HostnameMap parameter or from hostname if the parameter is empty.  In our deployment (newton + ContrailR4.0) we keep HostnameMap empty and this code works well.